Submitted By: Jim Gifford (patches at jg555 dot com)
Date: 2003-08-15
Initial Package Version: 0.50.3
Origin: http://www.mail-archive.com/aspell-user@gnu.org/msg00285.html
Description: Fixes GCC 3.3 Errors

diff -Naur aspell-0.50.3.orig/prog/checker_string.cpp aspell-0.50.3/prog/checker_string.cpp
--- aspell-0.50.3.orig/prog/checker_string.cpp	2002-09-26 02:42:42.000000000 +0000
+++ aspell-0.50.3/prog/checker_string.cpp	2003-08-15 23:20:21.000000000 +0000
@@ -9,6 +9,7 @@
 #include "document_checker.hpp"
 #include "copy_ptr-t.hpp"
 #include "asc_ctype.hpp"
+#include "assert.h"
 
 static int get_line(FILE * in, CharVector & d)
 {
